.github/renovate.json5

```diff
@@ -1,3 +1,16 @@
 {
-  "$schema": "https://docs.renovatebot.com/renovate-schema.json"
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
+  extends: [
+    "github>gradle/renovate-agent//presets/dv.json5"
+  ],
+  "packageRules": [
+    {
+      "description": "Use the last version of the Gradle Enterprise plugin and extension that does not provide the Develocity API",
```

Comment:
💬 `ignoreDeps` may be shorter but you can't provide a description for it, which seems like a nice thing to have.